docs/src/man/hosting.md

```diff
@@ -417,6 +417,37 @@ jobs:
 
 _This workflow was taken from [CliMA/ClimaTimeSteppers.jl](https://github.com/CliMA/ClimaTimeSteppers.jl/blob/0660ace688b4f4b8a86d3c459ab62ccf01d7ef31/.github/workflows/DocCleanup.yml) (Apache License 2.0)._
 
+## Woodpecker CI
+
+To run a documentation build from Woodpecker CI, one should create an access token
+from their forge of choice: GitHub, GitLab, or Codeberg (or any Gitea instance). 
+This access token should be added to Woodpecker CI as a secret named as
+`project_access_token` - all lowercase. Next, create a new pipeline configuration file
```

Comment:
Is this a convention or requirement in Woodpecker? You use `PROJECT_ACCESS_TOKEN` later, so is Woodpecker capitalizing the name? In that case, does it matter?

Reply:
Resolved in https://github.com/JuliaDocs/Documenter.jl/pull/1880/commits/f94f3a552b6c879c92a2163e468b29e1989e09c9

Reply:
If it gets passed uppercase anyway, maybe better to recommend setting declaring it uppercase as well, just to avoid this confusion?